Vic2ToHoI4/Source/V2World/World.cpp

```diff
@@ -517,4 +519,12 @@ void Vic2::World::checkAllProvincesMapped(const mappers::ProvinceMapper& provinc
 			Log(LogLevel::Warning) << "No mapping for Vic2 province " << province.first;
 		}
 	}
+}
+
+void Vic2::World::consolidateConquerStrategies()
+{
+	for (auto country: countries)
+	{
+		country.second->getAI()->consolidateConquerStrategies(provinces);
```

Comment:
Add a consolidateConquerStrategies() function to Country so you don't have to directly grab the AI